### ISSUE TITLE
purchase list - ability to filter on own vs other purchases  / add details

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-sdk#2921]
- [ ] should be able to pass a filter for your purchases vs someone purchasing your content 
- [ ] show more transaction details in output

**Comment by tzarebczan** [comment]
Per Lex's findings, purchase list only handles outgoing purchases and not incoming. What API do we want to use for incoming ones? If it's txo list, we need details added to the purchase.

**Comment by kauffj** [comment]
Yes, it should be `txo_list`. Ideally I could pass a claim id of a stream or channel and get the results accordingly.

**Comment by tzarebczan** [comment]
Also consider https://github.com/lbryio/lbry-sdk/issues/2939 as part of this. Not sure if this should be a new API or just part of resolve - probably the latter with a flag similar to other examples.

**Comment by kauffj** [comment]
Also for supports that are signed

**Comment by kauffj** [comment]
First call to develop is `support_xxx`, which will:
- Take a URL or claim id
- Sum the amount staked by identities, as well as amount staked by anonymous users

**Comment by kauffj** [comment]
Second is to develop APIs that would allow the creation of UIs that let me see my supports aggregated by claim and use `support_abandon` to manage the amounts.

**Comment by kauffj** [comment]
Summing supports historically has some challenges with regards to unlocking and history, @jackrobison will propose specific design. First step will be to get signing information into `txo_list`

**Comment by kauffj** [comment]
@lyoshenka wants to be involved in this, will discuss with you
